processor/schematransformerprocessor/testdata/config.yml

```diff
@@ -0,0 +1,29 @@
+receivers:
+  nop:
+
+processors:
+  schema_transformer:
+    # schema_url defines the remote location of the published
+    # schema translation to convert from an older semantic convention
```

Comment:
>to convert from an older semantic convention

I think we need to support converting both ways: from an older to newer and from newer to older.

Reply:
This also reads as an incomplete thought that didn't make through to the keyboard.
I'll also address this as well.